pkg/features/kube_features.go

```diff
@@ -1071,7 +1072,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	PodReadyToStartContainersCondition: {Default: true, PreRelease: featuregate.Beta},
 
-	PodHostIPs: {Default: true, PreRelease: featuregate.Beta},
+	PodHostIPs: {Default: true, PreRelease: featuregate.GA}, // GA in 1.30; Lock to default in 1.31; remove in 1.32
```

Comment:
we normally lock to default when it goes to GA... is there any reason to not do that here?

(and you can't remove the tag from the e2e test if it's still possible to disable the feature)

Reply:
Done